.openshift-ci/clusters.py

```diff
@@ -32,23 +32,26 @@ class GKECluster:
     REFRESH_PATH = "scripts/ci/gke.sh"
     TEARDOWN_PATH = "scripts/ci/gke.sh"
 
-    def __init__(self, cluster_id, num_nodes=3, machine_type="e2-standard-4", disk_gb=40):
+    def __init__(self, cluster_id, num_nodes=None, machine_type=None, disk_gb=None):
         self.cluster_id = cluster_id
         self.num_nodes = num_nodes
         self.machine_type = machine_type
         self.disk_gb = disk_gb
         self.refresh_token_cmd = None
 
     def provision(self):
+        if self.num_nodes is not None:
```

Comment:
Reportedly it's more pythonic to just do `if self.num_nodes:`.
Similarly below.